### PR TITLE
Center text support

### PR DESCRIPTION
**Pull request by ayushcshah**
Adding center align text

### CODE REVIEW

File: Pod/Classes/TSMessageView.m [hunk 1]
```diff
@@ -105,22 +99,21 @@ - (CGFloat)padding
...
-              image:(UIImage *)image
-               type:(TSMessageNotificationType)notificationType
-           duration:(CGFloat)duration
-   inViewController:(UIViewController *)viewController
-           callback:(void (^)())callback
-        buttonTitle:(NSString *)buttonTitle
-     buttonCallback:(void (^)())buttonCallback
-         atPosition:(TSMessageNotificationPosition)position
-canBeDismissedByUser:(BOOL)dismissingEnabled
+- (id)     initWithTitle:(NSString *)title
```
Comment (MichaelCampbell) [inline Pod/Classes/TSMessageView.m]: let's change this to instancetype

File: Pod/Classes/TSMessageView.m [hunk 2]
```diff
@@ -105,22 +99,21 @@ - (CGFloat)padding
...
+          buttonCallback:( void (^)() )buttonCallback
+              atPosition:(TSMessageNotificationPosition)position
+    canBeDismissedByUser:(BOOL)dismissingEnabled
 {
     NSDictionary *notificationDesign = [TSMessageView notificationDesign];
-    
-    if ((self = [self init]))
-    {
+
+    if ( (self = [self init]) ) {
```
Comment (MichaelCampbell) [inline Pod/Classes/TSMessageView.m]: We should refactor this so it matches the pattern

```
self = [super init];

if (self) {
...
}
```

File: Pod/Classes/TSMessageView.m [hunk 3]
```diff
@@ -129,80 +122,91 @@ - (id)initWithTitle:(NSString *)title
...
         CGFloat padding = [self padding];
-        
+
         NSDictionary *current;
         NSString *currentString;
-        switch (notificationType)
-        {
+        switch (notificationType) {
             case TSMessageNotificationTypeMessage:
             {
```
Comment (MichaelCampbell) [inline Pod/Classes/TSMessageView.m]: let's remove the curly braces between the case statements

File: Pod/Classes/TSMessageView.m [hunk 4]
```diff
@@ -129,80 +122,91 @@ - (id)initWithTitle:(NSString *)title
...
-        
+
         current = [notificationDesign valueForKey:currentString];
-        
-        
-        if (!image && [[current valueForKey:@"imageName"] length])
-        {
+
+
+        if (!image && [[current valueForKey:@"imageName"] length]) {
```
Comment (MichaelCampbell) [inline Pod/Classes/TSMessageView.m]: length is a property

File: Pod/Classes/TSMessageView.m [hunk 5]
```diff
@@ -254,40 +256,36 @@ - (id)initWithTitle:(NSString *)title
                                                   image.size.height);
             [self addSubview:self.iconImageView];
         }
-        
+
         // Set up button (if set)
-        if ([buttonTitle length])
-        {
+        if ([buttonTitle length]) {
```
Comment (MichaelCampbell) [inline Pod/Classes/TSMessageView.m]: length is a property, dot notation here as well

File: Pod/Classes/TSMessageView.m [hunk 6]
```diff
@@ -364,111 +356,117 @@ - (id)initWithTitle:(NSString *)title
     return self;
 }
 
+- (NSTextAlignment)textAlignmentValueForStyle:(NSDictionary *)dictionary
+{
+    static NSString *const kAATextAlignValueLeft = @"left";
+    static NSString *const kAATextAlignValueRight = @"right";
+    static NSString *const kAATextAlignValueCenter = @"center";
+
```
Comment (MichaelCampbell) [inline Pod/Classes/TSMessageView.m]: let's rename these constants from `kAA...` to `kTS...`

**Comment by ayushcshah** [comment]
Example is not showing up message properly with image

**Pull request closed without merging**